src/main/java/teamrtg/rtg/modules/bop/biomes/RTGBiomeBOPBayou.java

```diff
@@ -0,0 +1,86 @@
+package teamrtg.rtg.modules.bop.biomes;
+
+import biomesoplenty.api.biome.BOPBiomes;
+import net.minecraft.init.Biomes;
+import teamrtg.rtg.api.world.RTGWorld;
+import teamrtg.rtg.api.world.biome.TerrainBase;
+import teamrtg.rtg.api.world.biome.deco.DecoBaseBiomeDecorations;
+import teamrtg.rtg.api.world.biome.surface.part.CliffSelector;
+import teamrtg.rtg.api.world.biome.surface.part.SurfacePart;
+import teamrtg.rtg.modules.bop.RTGBiomeBOP;
+
+public class RTGBiomeBOPBayou extends RTGBiomeBOP {
+
+    public RTGBiomeBOPBayou() {
+        super(BOPBiomes.bayou.get(), Biomes.RIVER);
+    }
+
+    @Override
+    public TerrainBase initTerrain() {
+        return new TerrainBase() {
+
+            @Override
+            public float generateNoise(RTGWorld rtgWorld, int x, int y, float biomeWeight, float border, float river) {
+                return terrainPlains(x, y, rtgWorld.simplex, river, 80f, 1f, 40f, 20f, 62f);
+            }
+
+            /*
+            @Override
+            public float lakePressure(OpenSimplexNoise simplex, CellNoise simplexCell, int x, int y, float border)
```

Comment:
I wasn't sure where to put the water feature code, so I commented it out in the terrains.